data/conversations.txt

```diff
@@ -10,31 +10,33 @@
 	`	You sign your name one final time:`
 	name
-	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
-	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`
+	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. You bounce out of its metal chassis. A smile, ear to ear, carries you through the lobby, past the shining doormen, and out into the street. It is not until the end of the block that you finally jump and shout. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
+	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You hold hands with the smell of grease and dirt and rocket fuel as you drift towards the mechanic. When you ask her what she has in your price range she laughs. "I get one of you about every week." she explains, her face slick with sweat and engine grease. "Yeah alright let's take a look" With that, she leads you to a section of the shipyard with their more modest selections. As you walk, the ships surrounding you go from pristine, to adequate, to worn in, and the two of you finall end up in front of three seasoned vessels. She clasps her arm around your shoulder, chews, and spits. 
+
+"What'll it be, kid?"`
 
 conversation "flight check: no energy!"
 	scene "scene/engine2"
-	`You've heard enough horror stories about the sort of accidents that happen when starship systems malfunction to be completely convinced of the necessity of a pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of "<ship>" warming up, you run through a mental checklist. Computers: check. Life support: check.`
-	`	It is all going smoothly until it comes time to fire up your systems for a test run... and you discover that your generator does not produce enough power to power them. You'll have to either install some batteries, or switch to a more powerful generator.`	
+	`You have heard more than enough horror stories about the kinds of accidents that can happen when your starship systems malfunction and demand a non-negotiable pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of "<ship>" warming up, you run through a mental checklist. Computers: check. Life support: check.`
```

Comment:
That last part doesn't really fit here.

Reply:
could you specify what you mean by last part?

Reply:
`and demand a non-negotiable pre-flight safety check.`

It's only per societal standards to do that, it's not something quite something that the player himself would "demand". Yes, even at the risk of his life.

Reply:
I think I addressed it. I think I was just unclear as to who was demanding what from whom.